.azure/pipelines/jobs/default-build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       condition: ${{ parameters.condition }}
     ${{ if ne(parameters.enableRichCodeNavigation, '') }}:
       enableRichCodeNavigation: true
-      richCodeNavigationLanguage: 'csharp,typescript,java'
+      richCodeNavigationLanguage: 'csharp'
```

Comment:
This didn't change anything and https://dev.azure.com/dnceng/public/_build/results?buildId=1665654&view=results failed the same way as previous attempts. I suspect successful richnav pipelines are either all using `dotnet msbuild` exclusively or just have no C++ code in the repo. As a hack, could disable the native build entirely. Is that the right way forward❔

Reply:
It looks like you tried that but it still didn't work ☹️ Thanks for hacking away at this! We could also try filtering out the files via the trigger path fields like the runtime repo does [here](https://github.com/dotnet/runtime/commit/eb03e0f7bc396736c7ac59cf8f135d7c632860dd). We do successfully index other C++ repos though, so I'm hoping the package upgrade does the fix -- that PR is approved and awaiting merge [here](https://github.com/dotnet/arcade/pull/8619).